### ISSUE TITLE
Styling updates/fixes to docs site

### ISSUE THREAD

**Issue by wren** [jrnl-org/jrnl#1005]
## Documentation Change

After #1004 is merged, we should look over jrnl.rtfd.io to check for any readthedocs-specific css changes/fixes that might be needed.

### Affected Page(s)

  - https://jrnl.readthedocs.io/en/latest/
  - https://jrnl.sh/overview/

### What Could Be Better?

From what I could see in my early testing, here are a few things that already need fixing:
- [ ] "Read the Docs" bar is floating on right, instead of being nestled at the bottom of the sidebar (on the left)
- [ ] The little ad box in the sidebar might be pushed off the screen?
- [ ] The menu in the sidebar doesn't expand sub-menu items (this is also already a problem on [jrnl.sh](https://jrnl.sh))

### Other Information

There might be more things. We should do a sweep of the whole docs site and point out any other issues we find in this thread.

**Comment by wren** [comment]
I got an answer from RTD support about the index page problem:

> Hi Jonathan,
> 
> Thanks a lot for the explanation. Now, it's super clear the problem and I can confirm this is a Read the Docs issue.
> 
> We have been doing some magic to autogenerate a `conf.py` on Sphinx and README.rst or index.md on MkDocs when we detected the project was not fully configured to allow users to jump into the platform really quick. Now, this is bringing us some incompatibilities / issues with more advanced users with some customizations. In your case, you have a index that does not come from `index.md` but from a HTML file. So, Read the Docs detects there is no index.md and creates one for you --but you don't want that :)
> 
> We have some issues opened for this already and there is a PR to remove this magic as well:
> 
>     https://github.com/readthedocs/readthedocs.org/issues/2483​
>     https://github.com/readthedocs/readthedocs.org/pull/5609
>     https://github.com/readthedocs/readthedocs.org/issues/1800​
> 
> 
> However, we didn't have the time to make it review, make some design decisions and keep everything working without backwards compatibilities.
> 
> I just opened a new PR that adds a Feature flag to avoid this behavior of creating an index file that we could enable in your project to make it work as you want. I was able to build your documentation with your custom index by using my local RTD development instance with your project and this flag.
> 
> You can subscribe to that PR at https://github.com/readthedocs/readthedocs.org/pull/7305 to follow the discussion.
> 
> Regards,
> Manuel Kaufmann
> Read the Docs

**Comment by micahellison** [comment]
I think this fits in this issue:

- [ ] Fix the `!!! note` style (as seen in the External Editors page), which doesn't have a very distinct background so it's hard to tell when a "Note" ends.
